config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml

```diff
@@ -25,3 +25,21 @@ presubmits:
         - --
         - //...
         - -//vendor/...
+  - name: cloud-provider-gcp-e2e-create
+    max_concurrency: 5
+    decorate: true
+    path_alias: k8s.io/cloud-provider-gcp
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: provider-gcp-presubmits
+      description: End to end cluster creation test using cluster/kube-up based on kubernetes/cloud-provider-gcp.
+      testgrid-num-columns-recent: '30'
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200420-e830a3a-master
+        command:
+        - runner.sh
```

Comment:
is this using anything from runner? the service account?

Reply:
yes